__tests__/index.test.js

```diff
@@ -374,4 +374,39 @@ describe('DELETE /api/comments/:comment_id', () => {
         expect(body.msg).toEqual(expected);
       });
   });
+});
+describe('GET /api/users', () => {
+  it('GET 200: should respond with an array of objects, each object should have the following properties: username, name, avatar_url', () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const { body } = response;
+        const { users } = body;
+        const length = users.length;
+
+        expect(length).toBe(4);
```

Comment:
Great that you're checking the length to make sure we don't get a false positive if the user array were empty